src/components/VueMagnifier.vue

```diff
@@ -0,0 +1,202 @@
+<template>
+	<div>
+		<div class="vue-magnifier-container">
+			<slot></slot>
+			<span
+				ref="magnificationElement"
+				class="preview"
+				v-bind:style="{ backgroundImage: 'url(' + src + ')' }"
+			>
+				<span ref="glass" class="magnifying-glass" v-bind:style="glassStyle" />
+			</span>
+		</div>
+	</div>
+</template>
+
+<script>
+export default {
+	props: {
+		src: String,
+		srcLarge: String,
+	},
+	computed: {
+		glassStyle() {
+			return {
+				backgroundImage: `url(${this.srcLarge})`,
+				backgroundPosition: this.backgroundPos,
+				left: `${this.cursorX}px`,
+				top: this.cursorY + "px",
```

Comment:
why you are use the different string concut way please keep only single one I recommend `${this.cursorX}px` this one